tt_metal/hw/inc/ethernet/dataflow_api.h

```diff
@@ -50,14 +70,44 @@ volatile uint32_t *RtosTable =
     (volatile uint32_t *)&__erisc_jump_table;  // Rtos Jump Table. Runtime application needs rtos function handles.;
 
 FORCE_INLINE
-void reset_erisc_info() { erisc_info->user_buffer_bytes_sent = 0; }
+void reset_erisc_info() {
+    erisc_info->per_channel_user_bytes_send[0].bytes_sent = 0;
```

Comment:
I wouldn't mind if this was removed, don't think it's used anywhere

Reply:
is it not? In that case, yea, why not. I just populated these values based on the existing reset impl.

Reply:
I removed in the tunneling PR.